scripts/riscv_bootstrap.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+# This script will download and setup the riscv-linux required toolchain and tool.
+
+BOOTSTRAP_SCRIPT_PATH=$(dirname "$0")
+BOOTSTRAP_WORK_DIR=$BOOTSTRAP_SCRIPT_PATH/.bootstrap
+
+function cleanup {
+  if [ -d $BOOTSTRAP_WORK_DIR ]; then
+    rm -rf $BOOTSTRAP_WORK_DIR
+  fi
+}
+
+clear
```

Comment:
Delete `clear`? Any reason to have this?

Reply:
Sure. Just used to clean the screen.